DataFormats/JetReco/interface/PileupJetIdentifier.h

```diff
@@ -57,12 +58,26 @@ class StoredPileupJetIdentifier {
 	DECLARE_VARIABLE(ptD        ,float);
 
 	DECLARE_VARIABLE(nvtx   ,float);  
+	DECLARE_VARIABLE(rho   ,float);
```

Comment:
this will make changes proposed in https://github.com/cms-sw/cmssw/pull/10398
to be in conflict